app-text/cutemarked/cutemarked-0.11.3.recipe

```diff
@@ -0,0 +1,72 @@
+SUMMARY="Qt Markdown editor"
+DESCRIPTION="A Qt-based, free and open source Markdown editor with live HTML \
+preview, math expressions, code and markdown syntax highlighting."
+HOMEPAGE="https://cloose.github.io/CuteMarkEd"
+COPYRIGHT="2013 Christian Loose
+	2007 David Loren Parsons
+	2011-2012 Ali Rantakari
+	2012 CodeCatalyst, LLC"
+LICENSE="GNU GPL v2
+	MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/cloose/CuteMarkEd/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="a8f1c8f22ce52ff8a878607112a44ca3d45015120d07b301f78fc9631b894775"
+SOURCE_DIR="CuteMarkEd-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	cutemarked$secondaryArchSuffix = $portVersion
+	app:CuteMarkEd$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libhunspell_1.6$secondaryArchSuffix
+	lib:libmarkdown$secondaryArchSuffix
+	lib:libQt5WebKit$secondaryArchSuffix
+	lib:libQt5WebKitWidgets$secondaryArchSuffix
+	lib:libQt5Core$secondaryArchSuffix
+	lib:libQt5Gui$secondaryArchSuffix
+	lib:libQt5Widgets$secondaryArchSuffix
+	lib:libQt5Network$secondaryArchSuffix
+	lib:libQt5PrintSupport$secondaryArchSuffix
+	lib:libQt5Test$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libhunspell_1.6$secondaryArchSuffix
+	devel:libmarkdown$secondaryArchSuffix
+	devel:libQt5WebKit$secondaryArchSuffix
+	devel:libQt5WebKitWidgets$secondaryArchSuffix
+	devel:libQt5Core$secondaryArchSuffix
+	devel:libQt5Gui$secondaryArchSuffix
+	devel:libQt5Widgets$secondaryArchSuffix
+	devel:libQt5Network$secondaryArchSuffix
+	devel:libQt5QmlDebug$secondaryArchSuffix
+	devel:libQt5PrintSupport$secondaryArchSuffix
+	devel:libQt5Test$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:g++$secondaryArchSuffix
+	cmd:make
+	cmd:qmake$secondaryArchSuffix >= 5
+	cmd:lrelease$secondaryArchSuffix >= 5
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	qmake CuteMarkEd.pro
+	make $jobArgs
+}
+
+INSTALL()
+{
+	cutemarkedDir=$appsDir/CuteMarkEd/bin
+	mkdir -p $cutemarkedDir
+	cp $sourceDir/app/cutemarked $cutemarkedDir/CuteMarkEd
```

Comment:
If there is only one runtime, then you can copy it directly to $appsDir:
```diff
 INSTALL()
 {
-	cutemarkedDir=$appsDir/CuteMarkEd/bin
-	mkdir -p $cutemarkedDir
+	install -d $appsDir
-	cp $sourceDir/app/cutemarked $cutemarkedDir/CuteMarkEd
+	install -d $appsDir app/cutemarked
-
-	addAppDeskbarSymlink $cutemarkedDir/CuteMarkEd
+	addAppDeskbarSymlink $appsDir/cutemarked CuteMarkEd
 }
```

Reply:
Oops, I made a mistake on the second "install" of my previous suggestion. It should be:
```
	install -t $appsDir app/cutemarked
```

Reply:
I think "CuteMarkEd" is better name for app's binary. All files in $appsDir have capital letters in their names.

Reply:
No problem. In that case you can use this:
```
INSTALL()
{
	install -d $appsDir
	install -T app/cutemarked $appsDir/CuteMarkEd
	addAppDeskbarSymlink $appsDir/CuteMarkEd
}
```